tests/basic_store.py

```diff
@@ -20,8 +20,8 @@ def is_emulated_gcstore_test(store):
 
 class BasicStore(object):
     def test_store(self, store, key, value):
-        key = store.put(key, value)
-        assert isinstance(key, str)
+        new_key = store.put(key, value)
+        assert key == new_key
```

Comment:
I think I have seen some cases (in the redis store?) where you can have `new_key != key` is `key is None`. Should we handle that here, too?

Reply:
Here we return the `key`:

https://github.com/data-engineering-collective/minimalkv/blob/54cc3080d676bf0ea25a522263b3c992b7a40b56/minimalkv/memory/redisstore.py#L75-L97

I guess there it the `KeyTransformingDecorator` (undocumented): 
https://github.com/data-engineering-collective/minimalkv/blob/54cc3080d676bf0ea25a522263b3c992b7a40b56/minimalkv/decorator.py#L101-L102

and the `HashDecorator`:
https://github.com/data-engineering-collective/minimalkv/blob/54cc3080d676bf0ea25a522263b3c992b7a40b56/minimalkv/idgen.py#L49-L75

The former is surely untested (else tests would be failing).